### ISSUE TITLE
Decimals option ignored when format is "percentage"

### ISSUE THREAD

**Issue by beneuto** [metricsgraphics/metrics-graphics#794]
If format: "percentage" option is set then the data rollover will show percentage to 6 decimal places regardless of any value assigned to the decimals option.

**Comment by wlach** [comment]
I can't actually reproduce this, are you sure?

**Comment by beneuto** [comment]
Yep, see screenshot attached.

From code:

```
          MG.data_graphic({
            data: data,
            top: 20,
            full_height: true,
            full_width: true,
            target: document.getElementById("factor"+scope),
            x_accessor: "timestamp",
            y_accessor: "power_factor",
            format: "percentage",
            decimals: 0,
            min_y: 0,
            missing_is_hidden: true,
            legend: legend,
            legend_target: '.legend',
            transition_on_update: false
            });
```


![image](https://user-images.githubusercontent.com/25493799/36011130-1845c31e-0da1-11e8-8fca-6287476fad4b.png)

**Comment by wlach** [comment]
@beneuto ok, I see that if I set decimals to 0, we see a bunch of decimals. Is that the issue you're seeing? Because otherwise this feature works as expected. 

Just want to make sure we understand the problem before trying to fix it. :)

**Comment by beneuto** [comment]
Yep, that's it.

**Comment by wlach** [comment]
This would make a decent good first bug, you should be able to reproduce it in the development environment described here:

https://github.com/mozilla/metrics-graphics#contributing

Then git grep for "percentage" in the source and try to find the source of the problem.

**Comment by Rupeshiya** [comment]
@wlah How can I fix this issue ?

**Comment by cnwangjie** [comment]
@wlach I fixed it in #901
